scripts/show-features.py

```diff
@@ -199,6 +199,21 @@ def ida_main():
     return 0
 
 
+def ghidra_main():
+    import capa.features.extractors.ghidra.extractor
+
+    extractor = capa.features.extractors.ghidra.extractor.GhidraFeatureExtractor()
+
+    for feature, addr in extractor.extract_file_features():
+        print(f"file: {format_address(addr)}: {feature}")
+
+    function_handles = tuple(extractor.get_functions())
+
+    print_features(function_handles, extractor)
+
+    return 0
```

Comment:
Enables `show-features.py` to be run from Ghidra headless/UI

Reply:
Very useful!